tests/baselines/reference/uniqueSymbols.types

```diff
@@ -524,10 +524,10 @@ const o2 = {
 >s : unique symbol
 
     method5(p = s) { return p; },
->method5 : (p?: symbol) => symbol
->p : symbol
+>method5 : (p?: unique symbol) => symbol

```

Comment:
These kinds of changes don't make me happy and probably indicate that this change is probably not good as-is.